xml/Microsoft.VisualBasic.Vsa/VsaEngine.xml

```diff
@@ -431,7 +432,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets a value that indicates whether the current in-memory representation of the item differs from the persisted representation.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the script engine is marked as modified (that is, "dirty"); otherwise, <see langword="false" />. The default value is <see langword="false" />.</value>
```